ci-operator/step-registry/openshift-qe/cluster-density/OWNERS

```diff
@@ -1 +0,0 @@
-../OWNERS
```

Comment:
Add this file back and it all looks good to me!!

Reply:
Thank you for taking time review my PR